assets/eip-3475/Debond-ERC3475.sol

```diff
@@ -1,18 +1,31 @@
-// SPDX-License-Identifier: CC0-1.0
+// SPDX-License-Identifier: apache 2.0
+/*
+    Copyright 2022 Debond Protocol <info@debond.org>
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+*/
 
 pragma solidity ^0.8.0;
 
-// @dev install the dependencies 
-// npm i debond-erc3475-contracts.
 
-import "debond-erc3475-contracts/interfaces/IDebondBond.sol";
-import "debond-erc3475-contracts/interfaces/IRedeemableBondCalculator.sol";
-import "debond-governance-contracts/utils/GovernanceOwnable.sol";
+import "./interfaces/IDebondBond.sol";
+import "./interfaces/IProgressCalculator.sol";
+import "@debond-protocol/debond-governance-contracts/utils/GovernanceOwnable.sol";
+
 
 contract DebondERC3475 is IDebondBond, GovernanceOwnable {
-    address bankAddress;    
+
+    address bankAddress;
+
     /**
-    * @notice this Struct is representing the Nonce properties as an object
+    * @notice  Here struct is representing the Nonce properties as an object
```

Comment:
I'm not sure why you are changing this, `this struct` is more correct I believe.  Plus single space.

Reply:
sure my bad. Actually this was the last check  I did via Grammarly

Reply:
this implementation is removed